### ISSUE TITLE
Ability to translate the user biography

### ISSUE THREAD

**Issue by nekohayo** [maximeschoeni/sublanguage#44]
The various fields in wp-admin/profile.php don't seem to be translateable. It would be nice if we could have a different publicly-visible name, title, company name, biography, etc. for use with plugins such as https://wordpress.org/plugins/starbox/

**Comment by maximeschoeni** [comment]
Right, I never needed this feature so I never implemented it. Maybe in the future.

**Comment by okainov** [comment]
Would be great to have it
